### ISSUE TITLE
Detection of new professions

### ISSUE THREAD

**Issue by JaneX8** [OlafenwaMoses/IdenProf#2]
I'm thinking of new professions to add to this project. Since professions can be dressed differently in different countries and cultures this is hard. But I think that a religious "leader" such a priests, pastor, father, imam, monk, bishop, preacher, rabbi are quite identifiable. Is there any interest in this? I could help collect those images.

**Comment by OlafenwaMoses** [comment]
We are open to adding more classes and images to the dataset. We only want to ensure enough images per class are collected (at least 1000 unique images) . You can reach to my email via guymodscientist@gmail.com if you want to discuss more and contribute classes and images to this dataset.

**Comment by ElleshaHackett** [comment]
Hi Moses,
Thanks for your reply. I'm new to machine learning although I have plenty of knowledge in several programming languages.

I've started tonight by making a photo collection of soldiers. I've currently selected and deduplicated my collection and like to share it with you soon. Some photos are high quality. I've noticed in your training set that all images are resized without maintaining ratio. Can you tell me the prerequirements for a good collection so I can improve the set before sharing it with you. Can their be only one or a few people or also a group of people? What about the size and image format?

Also, can you maybe supply a little bit more information what is needed in order to get this training and testing started?

I look forward to help your project forward and to share knowledge to improve this project and our skills.

Greetings,
Kevin Morssink
The Netherlands
________________________________

**Comment by JaneX8** [comment]
I've send you an email

**Comment by JaneX8** [comment]
@OlafenwaMoses I have four questions:

1. You wrote "This pre-trained model was trained over 61 epochs only, but it achieved 79% accuracy on 2000 test image" Why only 61 epochs? Do you need resources/hardware to get a higher accuracy?

2. You wrote "More experiments will enhance the accuracy of the model.". Can you explain what kind of experiments you have in mind and are you planning to perform those experiments in the near future to improve accuracy?

3. The images for training are all resized down to a low quality format and have not remained the original ratio aspects. Is this normal? Wouldn't the result be higher with higher quality images with original aspect ratio? (Do you also have the original photos from before resizing and scaling?)

4. In order to add images. What are the requirements for that? Certain file format, aspect ratios, file size, image size? Can there be only one person on the photo or multiple (group)? Should the face be always visible or should it be a photo's from the back or side too? Can the photo have (a small) watermark or side note? Can it be a (digital) drawing representing the same profession too? I've started collecting a set of soldiers. Which is probably not ideal yet. Please some feedback on the following collection:
`https://drive.google.com/uc?id=1Beyjtg9SMAKaXVmmR-24v5KA7wFvQ91x&export=download`.

_By the way I am testing your script now (training) on Windows and Ubuntu._
